src/configurations/destinations/http/ui-config.json

```diff
@@ -541,6 +544,12 @@
           {
             "name": "Request Body",
             "fields": [
+              {
+                "type": "checkbox",
+                "label": "Send the event payload as it is",
+                "configKey": "isDefaultMapping",
+                "default": true
```

Comment:
these are part of advanced config?

Reply:
I didn't get you Dilip. what do you mean by advanced config?